syntax = "proto2";
package cardgame;

import "struct.proto";
option optimize_for = LITE_RUNTIME;
option java_package = "com.zhili.cardgame.proto";  
option java_outer_classname = "Bag";


message QueryBagReq {
	enum DEF {
		ID = 4200;
	}
	
}

message QueryBagResp {
	enum DEF {
		ID = 4201;
	}
	repeated Item items = 1;
	repeated Gift gifts = 2;
}

	

